todo_app/app.py

```diff
@@ -8,4 +10,26 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    items = sorted(items, key=lambda item: item['status'])
+    return render_template('index.html', items=items)
+
+@app.route('/add', methods = ['POST'])
+def addItem():
+    title = request.form.get('title')
+    add_item(title)
+    return redirect('/')
+
+@app.route('/delete', methods = ['POST'])
+def deleteItem():
+    item_id = request.form.get('item_id')
+    delete_item(item_id)
+    return redirect('/')
+
+@app.route('/markComplete', methods = ['POST'])
+def markItemAsComplete():
+    item_id = request.form.get('item_id')
+    item = get_item(item_id)
+    item['status'] = 'Completed'
```

Comment:
minor, but could be cleaner to use enums rather than strings to compare statuses